apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/SnifferConfigInitializer.java

```diff
@@ -207,4 +212,15 @@ private static InputStreamReader loadConfig() throws AgentPackageNotFoundExcepti
         }
         throw new ConfigNotFoundException("Failed to load agent.config.");
     }
+
+    static void configureLogger() {
```

Comment:
This is not a good way to do this switch. I am waiting for the PMC confirmation about whether JSON format is acceptable. If YES, we need to improve this.